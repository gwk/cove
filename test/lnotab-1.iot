{
  'cmd': '$PROJ/cove.py test/lnotab.py 1',
}
