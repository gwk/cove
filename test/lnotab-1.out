cove report:

__main__: test/lnotab-{}.py
[30m   4 [37m  [0m
[30m   5 [0m  x = ''[0m
[30m   6 [0m  def top(a):[0m
[30m   7 [37m    global x[0m
[30m   8 [31m%   while a:[0m
[30m   9 [0m      x = a[0m
[30m  10 [0m      break[0m
[30m  11 [37m    else:[0m
[30m  12 [31m!     x = a[0m
[30m  13 [37m  [0m

__main__: test/lnotab-{}.py: 18 lines; [37m7 trivial[0m; 11 traceable; 8 covered; 0 relaxed; [36m1 ignored[0m; 0 ignored but covered; [31m2 not covered[0m.
